README.md

```diff
@@ -51,6 +51,11 @@ sequences into their byte representations.
 **note** if an error is thrown from this function, the original, non-decoded cookie value will
 be returned as the cookie's value.
 
+##### multiValuedCookies
+
+Specifies a boolean flag which if set to `true` would allow reading multiple cookies with same name (key). All values will be put into an array.
+By default (when `multiValuedCookies` isn't present or set to `false`), only first cookie's value is read which is not in accordance with [RFC 6265](https://tools.ietf.org/html/rfc6265#section-4.2.2) as discussed [here](https://github.com/jshttp/cookie/issues/60) (this behaviour is kept around for backward compatibility reason).
```

Comment:
I would remove the issue reference from the docs. If there is some information in there that is useful, add it directly in the docs.

Reply:
https://www.baoding68d.net/list-games?category=

Reply:
https://www.baoding68d.net/list-games?category=